style.css

```diff
@@ -396,3 +404,145 @@ form button:hover:active{
   text-align: center;
   padding-bottom: 30px;
 }
+
+
+/* -------------------Media queries------------------ */
+
+@media screen and (min-width : 992px){
+  body{
+    width: 100%;
+    display: flex;
+  }
+
+  .main-wrapper{
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-self: center;
+    background-color: #f7f7f9;
+  }
+  .wrapper {
+    /* background-repeat: no-repeat; */
+    background-size: 100%;
+    width: 100%;
+    margin: 0;
+    align-self: center;
+    border-radius: 0px 0px 0px 90px;
+  }
```

Comment:
As the previous reviewer asked  
- [ ] Kindly uncomment the `/* background-repeat: no-repeat; */`, because you need it.

- [ ] Kindly remove the `background-color: white;` from the header section. to show the background image. It will remove the color from the about section but you can apply the the  `background-color: white;` only for the about section.

https://github.com/AnicetFantomas/setup-and-mobile-version-skeleton/blob/9ff4e17cb70552784c7143ff2528c5f2daf38509/style.css#L234-L237

![image](https://user-images.githubusercontent.com/35535130/155538655-e590e202-d6e8-4615-be0e-1b51eff5c3cb.png)